app/assets/javascripts/webchat.js.erb

```diff
@@ -25,6 +25,7 @@
         '/vat-registration/changes-to-your-details',
         '/government/organisations/hm-revenue-customs/contact/national-insurance-numbers',
         '/lost-national-insurance-number',
+        '/manage-your-tax-credits',
```

Comment:
We tend to keep trailing commas around to reduce noisy diffs when the next change comes (otherwise you'd have two lines changed because of adding a comma to the previous line) but I suppose so, if it breaks something...